recipes/merqury/meta.yaml

```diff
@@ -1,30 +1,29 @@
 {% set name = "merqury" %}
-{% set version = "1.1" %}
-{% set sha256 = "2140d87d9fa7a4a251905737cd96481fc9dfe3f3f4dd92c91adcce051df75cfe" %}
+{% set version = "1.3" %}
+{% set sha256 = "b6e3f54a6fe30acfe6f2b15ebc26a103a51980adfc03522e712a2baa5e79131a" %}
 
 package:
   name: "{{ name }}"
   version: "{{ version }}"
 
 source:
-  url: https://github.com/marbl/{{ name }}/archive/v{{ version }}.tar.gz
+  url: https://github.com/marbl/{{name}}/archive/refs/tags/v{{version}}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 0
   noarch: generic
 
 requirements:
   run:
-    - meryl  >=v1.0,<2000
+    - meryl  >=v1.3,<2000
```

Comment:
should this be pinned to 1.3 strict? Do you expect that the this version of merqury will work with the next versions of meryl?

Reply:
yeah, you're probably right